sink/datadog.go

```diff
@@ -18,25 +20,28 @@ import (
 	"github.com/cashapp/blip/status"
 )
 
-// SignalFx sends metrics to SignalFx.
+const DefaultDogStatsDPort = 8125
+
+// Datadog sends metrics to Datadog.
 type Datadog struct {
 	monitorId string
 	tags      []string            // monitor.tags (dimensions)
 	tr        tr.DomainTranslator // datadog.metric-translator
 	prefix    string              // datadog.metric-prefix
-	// --
+
+	// -- Api
 	metricsApi *datadogV2.MetricsApi
 	apiKeyAuth string
 	appKeyAuth string
+	resources  []datadogV2.MetricResource
 
-	resources []datadogV2.MetricResource
+	// -- DogStatsD
+	dogStatsD       bool
+	dogStatsDClient *statsd.Client
+	dogStatsDHost   string
```

Comment:
Generally you're correct to Caps a thing however the source does, and in this case I see DD does spell it like "DogStatsD". But since these are internal/private fields, the simpler `dogstatsd` is probably ok because I bet... yep, even in their own Go code example they lowercase it:
```go
dogstatsd_client, err := statsd.New("127.0.0.1:8125")
```

Reply:
Will update 👍